src/app/components/plugin-manager-component.js

```diff
@@ -37,12 +47,15 @@ const css = csjs`
     color: 
   }
   .versionWarning {
-    background-color: var(--light);
-    padding: 0 7px;
-    font-weight: bolder;
-    margin-top: 5px;
-    text-transform: lowercase;
+    padding: 4px;
+    margin: 0 8px;
+    font-weight: 700;
+    font-size: 9px;
+    line-height: 12px;
+    text-transform: uppercase;
```

Comment:
Would this be better as a badge? - I realize that it is currently coded with .versionWarning.

Reply:
Made it as .badge-light. Check it please

Reply:
Sorry - I don't see any .badge-light on plugin-manager-component.js
Please point me to where you changed it.  

Also I think there will be some design adjustment to this page - I'll be reviewing that with Natalia on Thursday.